test/integration/targets/nxos_system/tests/cli/sanity.yaml

```diff
@@ -0,0 +1,33 @@
+- debug: msg="START TRANSPORT:CLI nxos_system sanity test"
+
+- block:
+  - name: configure hostname and domain-name
+    nxos_system: &hostname
+      hostname: "{{ inventory_hostname_short }}"
+      domain_name: test.example.com
+      provider: "{{ cli }}"
+
+  - name: remove configuration
+    nxos_system:
+      state: absent
+      provider: "{{ cli }}"
+
+  - name: configure name servers
+    nxos_system:
+      name_servers:
+        - 8.8.8.8
+        - 8.8.4.4
+      provider: "{{ cli }}"
+
+  - name: configure name servers with VRF support
+    nxos_system:
+      name_servers:
+        - { server: 8.8.8.8, vrf: management }
+        - { server: 8.8.4.4, vrf: management }
+      provider: "{{ cli }}"
+
+  always:
+  - name: Re-configure hostname
+    nxos_system: *hostname
+
+  - debug: msg="END TRANSPORT:CLI nxos_system sanity test"
```

Comment:
Please update the debug statement to
`- debug: msg="END connection={{ ansible_connection }}/sanity.yaml"`